docs/version_3.md

```diff
@@ -155,7 +155,7 @@ If you have custom formats you can make use of this feature too! The `dictionary
 ```javascript
 StyleDictionary.registerFormat({
   name: `myCustomFormat`,
-  formatter: function(dictionary) {
+  formatter: function({ dictionary }) {
     return dictionary.allProperties.map(token => {
```

Comment:
I know this is our mistake, but can you change "allProperties" to "allTokens"?

Reply:
Sure thing. 🙂